diff --git a/dist/loaders/Item.js b/dist/loaders/Item.js
index 56f341650c7c37d52d0093a6f6f9261132620407..f6e64705c8adf1683daa8fea12e266f3f01fb6d6 100644
--- a/dist/loaders/Item.js
+++ b/dist/loaders/Item.js
@@ -291,6 +291,10 @@ export class Item extends PerFileLoadable {
                 case 149:
                     v.placeholderTemplate = r.u16();
                     break;
+                case 160:
+                    v.isStackable = false;
+                    v.hasVar = true;
+                    break;
                 case 200:
                     v.groundOps.decodeSubOp(r);
                     break;
